tools/lib/utils.coffee

```diff
@@ -0,0 +1,61 @@
+grep  = require('grepit')
+fs = require('fs')
+
+CHECKSUM_FILE_PATH = 'SHASUMS256.txt'
+UNSUPPORTED = 'UNSUPPORTED'
+
+ALPINE_ARCH =
+	'amd64': 'amd64'
+	'i386': 'i386'
+	'armv6hf': 'armhf'
+	'armv7hf': 'armhf'
+
+FEDORA_ARCH = 
+	'armv7hf': 'armv7hf'
+
+BUILDPACK_TEMPLATE = 
+
+
+resolveArch = (arch, distro) ->
+	if distro is 'alpine'
+		return ALPINE_ARCH[arch] or UNSUPPORTED
+	if distro is 'fedora'
+		return FEDORA_ARCH[arch] or UNSUPPORTED
+	return arch
+
+exports.getChecksum = (filename) ->
+	result = grep(filename, CHECKSUM_FILE_PATH)
```

Comment:
@nghiant2710 this would be a lot better if the checksum file was JSON and it properly parsed it instead of grepping